app/controllers/application_controller.rb

```diff
@@ -521,6 +523,21 @@ def user_is_banned?
     BannedUserResolver.new(current_user).banned_for_sp?(issuer: current_sp&.issuer)
   end
 
+  def user_account_creation_device_profile_failed?
+    return unless IdentityConfig.store.account_creation_device_profiling == :enabled
+    profiling_result = find_device_profiling_result(
+      DeviceProfilingResult::PROFILING_TYPES[:account_creation],
+    )
+    profiling_result&.rejected?
```

Comment:
It is not clear what `success` vs `review_status` is in `DeviceProfilingResult`. Also, what is the difference between `failed` and `rejected`?

Reply:
success Is simple of what we think is a success vs failure. But I think review_status is a little bit more involved/ in depth than the success attribute.

Reply:
If `success` can be derived from `review_status`, we should only persist `review_status` and leave `success` as a method on the model.

Reply:
removed success as well